_docker/drupal/Dockerfile.mp

```diff
@@ -24,12 +24,19 @@ COPY drupal-filesystem/web/sites/default /var/www/drupal/web/sites/default
 COPY drupal-filesystem/web/modules/custom /var/www/drupal/web/modules/custom
 COPY drupal-filesystem/web/themes/custom /var/www/drupal/web/themes/custom
 
-# Build the theme
+# Build the rhdp theme
 RUN cd /var/www/drupal/web/themes/custom/rhdp/rhd-frontend \
     && npm install \
     && npm run-script build \
     && rm -rf /var/www/drupal/web/themes/custom/rhdp/rhd-frontend/node_modules
 
+# Build the rhdp2 theme
```

Comment:
@robpblake This is the PR, and the specific part that fails, that we discussed today. If I uncomment this block/command and execute the build, it completes successfully. I then have to pull the rhd-frontend git module, cd into the directory, run `npm install`, run `npm run styles`, run `npm run build` and then I have the `dist` directory with the dist assets as intended. This RUN command will fail though.

We need some way to make this RUN command execute successfully or some command that yields our desired end state of the dist assets being built out in the `dist` directory within `rhd-frontend`. I think that one thing, and there may be others, that current causes this to fail is the new method of including the FortAwesome depenendenc(ies). @KamiQuasi suggested the following:

> we would just commit the .npmrc file with this:
```@fortawesome:registry=https://npm.fontawesome.com/
//npm.fontawesome.com/:_authToken=${FONTAWESOME_NPM_AUTH_TOKEN}```

where `${FONTAWESOME_NPM_AUTH_TOKEN}` is an env variable.

Reply:
@jordanpagewhite As discussed yesterday, I'll work on a temporary solution that lets us build this as part of the Docker build until the CDN work is ready. Please keep the CDN stuff in the back of your mind when doing the integration with the new `rhd-frontend`. 

Standby for a PR to fix this issue.